src/snapshot/benches/main.rs

```diff
@@ -8,6 +8,7 @@ use versionize_derive::Versionize;
 mod version_map;
 
 #[derive(Clone, Debug, Default, Versionize)]
+// NOTICE: Any changes to this structure require a snapshot version bump.
```

Comment:
This is not needed for structure used in tests. Changing these structures will become mandatory anyway after any structures used in non-test code source since we expect that the tests will fail if they are not adjusted properly.

Reply:
@dianpopa done.